frontend/src/scenes/authentication/signup/SignupContainer.tsx

```diff
@@ -50,51 +56,82 @@ export function SignupContainer(): JSX.Element | null {
     ) : null
 }
 
+type ProductBenefit = {
+    benefit: string
+    description: string | ReactNode
+}
+
+const getProductBenefits = (featureFlags: FeatureFlagsSet): ProductBenefit[] => {
+    const signupBenefitsFlag = featureFlags[FEATURE_FLAGS.SIGNUP_BENEFITS]
+    switch (signupBenefitsFlag) {
+        case 'generic-language':
+            return [
+                {
+                    benefit: 'Free usage every month - even on paid plans',
+                    description: '1M free events, 5K free session recordings, and more. Every month. Forever.',
+                },
+                {
+                    benefit: 'Start collecting data immediately',
+                    description: 'Integrate with developer-friendly APIs or low-code web snippet.',
+                },
+                {
+                    benefit: 'Join industry leaders that run on PostHog',
+                    description:
+                        'Airbus, Hasura, Y Combinator, Staples, and thousands more trust PostHog as their Product OS.',
+                },
+            ]
+        case 'logos':
+            return [
+                {
+                    benefit: 'Free for 1M events every month',
```

Comment:
Is `1M events free every month` more clear?

Reply:
Yep, I like that more 👍